src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicAnalysisResourceTests.cs

```diff
@@ -44,7 +44,7 @@ public static void FlushPayload()
 ";
 
             var c = CreateCompilationWithMscorlib(Parse(source, @"C:\myproject\doc1.cs"));
-            var peImage = c.EmitToArray(EmitOptions.Default.WithEmitDynamicAnalysisData(true));
+            var peImage = c.EmitToArray(EmitOptions.Default.WithInstrument("Test.Flag"));
```

Comment:
Should we have a test to verify that instrumentation spans are not added to the IL when the "instrument" flag is not present?